http-gateway/swagger.yaml

```diff
@@ -751,6 +751,9 @@ components:
           description: JWT claim identifying device owner.
           type: string
           default: 'sub'
+        jwtDeviceIdClaim:
+          description: JWT claim identifying device id. Empty means that JWT doesn't contains it.
```

Comment:
typo: JWT doesn't contains it -> JWT doesn't contain it

Reply:
Fixed